modules/gathering-operator-logs.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting-operator-issues.adoc
+
+[id="gathering-operator-logs_{context}"]
+= Gathering Operator logs
+
+Gather detailed diagnostic information from Operator Pod logs.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Your API service is still functional.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* You have the fully-qualified domain names of the control plane, or master, machines.
+
+.Procedure
+
+. List Operator Pods running in the Operator's namespace, plus Pod status, restarts and age:
```

Comment:
Also, +1 to @bobfuru's mentions of using the serial (or Oxford) comma when you're listing things. Like "x, y, and z" instead of "x, y and z". It can be writer preference, but I think it leads to better clarity in the docs.

Reply:
Thank you. I will factor this into my writing in future. I have now updated this line.